src/main/java/org/jitsi/videobridge/LipSyncHack.java

```diff
@@ -318,7 +325,31 @@ void onRTPTranslatorWillWriteAudio(
      * @param sz the number of black key frame packets to create.
      * @return the array of black key frame packets that were created.
      */
-    private static RawPacket[] make(int ssrc, int seqNumOff, long tsOff, int sz)
+    private RawPacket[] make(int ssrc, int seqNumOff, long tsOff, int sz)
```

Comment:
is it not possible/practical to pull these from signaling instead?  not that this wouldn't work, but technically we should be following what has been signaled, right?  might also make things more straightforward if we actively don't accept media for payload types that haven't been signaled.